control-plane/api-gateway/common/translation_test.go

```diff
@@ -1267,7 +1272,21 @@ func TestTranslator_ToHTTPRoute(t *testing.T) {
 										ExtensionRef: &gwv1beta1.LocalObjectReference{
 											Name:  "test",
 											Kind:  v1alpha1.RouteRetryFilterKind,
-											Group: "consul.hashicorp.com/v1alpha1",
+											Group: ConsulHashicorpGroup,
```

Comment:
Since `github.com/hashicorp/consul-k8s/control-plane/api/v1alpha1` is already imported, I think you can just do something like this to avoid redefining the constant everywhere:

```suggestion
											Group: v1alpha1.GroupVersion.String(),
```

Same with the other places where this is used.